config/core/test-configs.yaml

```diff
@@ -2366,6 +2378,16 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: kontron-kswitch-d10-mmt-6g-2gs
+    test_plans:
+      - baseline
+      - baseline-nfs
+
+  - device_type: kontron-kswitch-d10-mmt-8g
```

Comment:
Not online yet, will check again once it has been added to the lab.

Reply:
@mwalle This one is still not online. If you split the PR to only include `kontron-kswitch-d10-mmt-6g-2gs` it can be merged tomorrow, then the `-8g` variant could be added later once it's available in the lab.